apps/test/storybook/util/testStorybook.js

```diff
@@ -8,13 +8,35 @@ import {withReduxStore} from '../../util/withReduxStore';
  * warnings.
  * @param {function(Storybook)} storiesFn
  */
-export default function testStorybook(storiesFn) {
-  const testBook = new FakeStorybook();
-  storiesFn(testBook);
-  testBook.test();
+export default function testStorybook(storyFile) {
+  // The deprecated `storiesOf` format exports stories as a single function.
+  // The new CSF format exports stories as an object.
+  if (typeof storyFile === 'function') {
+    const testBook = new FakeStorybook_DEPRECATED();
+    storyFile(testBook);
+    testBook.test(storyFile);
+  } else {
+    testStorybookFile(storyFile);
+  }
+}
+
+function testStorybookFile(file) {
+  const {default: defaultExport, ...stories} = file;
```

Comment:
What is `defaultExport`? A JS thing, Storybook?

Reply:
just JS. you can access the default export from the file object this way, but you can't name a variable named "default," so i'm renaming it here